internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/kafka/KafkaConsumptionTest.java

```diff
@@ -183,9 +188,14 @@ public void testLocalAndRemoteConsumption(KafkaConsumerService.ConsumerAssignmen
         new OptimizedKafkaValueSerializer(),
         new LandFillObjectPool<>(KafkaMessageEnvelope::new),
         new LandFillObjectPool<>(KafkaMessageEnvelope::new));
+
+    Properties pubSubProps = new Properties();
+    pubSubProps.putAll(pubSubClientProperties);
+    VeniceProperties veniceProperties = new VeniceProperties(pubSubProps);
```

Comment:
you can simply do 
`VeniceProperties veniceProperties = new VeniceProperties(pubSubClientProperties);`
Strings are immutable, so you dont have to worry about change of values outside.

Reply:
We don't have `public VeniceProperties(Map<String, String> properties) {` constructor but I suppose we can add one